service-catalog/gcp-backup/README.md

```diff
@@ -0,0 +1,132 @@
+# GCP backup
+
+## Summary
+
+In GCP (Google Cloud Platform) we keep offsite backups for both Rust releases and crates
+to protect us against security threats that could involve losing crates or releases.
+These threats were identified in the [threat model].
```

Comment:
```suggestion
These threats were identified in a [threat model] for the project's infrastructure, created by the Rust Foundation's security engineer Walter.
```

There are multiple threat models, and I think it's be nice to call out @walterhpearce who created them. 🙂